src/modules/params/greeting.rs

```diff
@@ -1,17 +1,18 @@
 use anyhow::Result;
 
 pub fn get(args_toggle_greeting: bool, config_greet: Option<bool>) -> Result<bool> {
-	if !args_toggle_greeting && config_greet.is_none() {
-		return Ok(true);
```

Comment:
In fact with the requested changes below we wouldn't even need the guard clause anymore. This would slim it down further. So no need to revert this removal.